coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = vars(result)
```

Comment:
You are overwriting the `args` variable which is passed to this enclosing function. which is bad